index.bs

```diff
@@ -8,14 +8,20 @@ Editor: Michael Blain, Google, mpb@google.com
 Abstract: This specification describes  an API that can be used to understand the visibility and position of DOM elements relative to a viewport. The position is delivered asynchronously and is useful for understanding the visibility of elements and implementing pre-loading and deferred loading of DOM content.
 Group: Web Performance Working Group
 Repository: slightlyoff/IntersectionObserver
-Mailing List: public-web-perf@w3c.org
-Mailing List Archives: http://lists.w3.org/Archives/Public/public-web-perf/
 </pre>
 
 <pre class="anchors">
-text: DOMHighResTimeStamp; type: typedef; urlPrefix: http://www.w3.org/TR/hr-time/
-text: margin; type: attribute; url: http://www.w3.org/TR/CSS21/box.html#propdef-margin
-text: length; type: attribute; url: http://www.w3.org/TR/css3-values/#lengths
+urlPrefix: http://www.w3.org/TR/hr-time/; type: typedef; text: DOMHighResTimeStamp
+url: http://www.w3.org/TR/CSS21/box.html#propdef-margin; type: attribute; text: margin
+url: http://www.w3.org/TR/css3-values/#lengths; type: attribute; text: length
+urlPrefix: http://www.w3.org/TR/html5/browsers.html; type: dfn; text: unit of related similar-origin browsing contexts
+url:http://w3c.github.io/requestidlecallback/#dfn-list-of-idle-request-callbacks; type: dfn; text: list of idle request callbacks
+url:http://heycam.github.io/webidl/#dfn-callback-this-value; type: dfn; text: callback this value
+urlPrefix:http://www.w3.org/TR/html51/webappapis.html; type: dfn; text: report the exception
+url:http://www.w3.org/TR/2014/WD-css-masking-1-20140213/#the-clip-path; type: dfn; text: clip path
+url:https://dom.spec.whatwg.org/#concept-node-remove-ext; type: dfn; text: removing steps
+url:https://dom.spec.whatwg.org/#concept-tree-inclusive-descendant; type: dfn; text: inclusive descendants
+url:https://dom.spec.whatwg.org/#concept-tree-order; type: dfn; text: tree order
```

Comment:
Ah. removing steps, inclusive descendants and tree order actually do seem to work. I think callback this value and report the exception didn't.